planetiler-custommap/samples/owg_simple.yml

```diff
@@ -0,0 +1,192 @@
+schemaName: OWG Simple Schema
+schemaDescription: Simple vector tile schema
+attribution: <a href="https://www.openstreetmap.org/copyright" target="_blank">&copy;
+  OpenStreetMap contributors</a>
+sources:
+- name: water_polygons
+  type: shapefile
+  url: https://osmdata.openstreetmap.de/download/water-polygons-split-3857.zip
+- name: osm
+  type: osm
+  area: geofabrik:rhode-island
+layers:
+- name: water
+  features:
+  - sources:
+    - osm
+    zoom:
+      minZoom: 0
+      maxZoom: 14
+    includeWhen:
+      geometry: polygon
+      tag:
+        key: natural
+        value:
+        - water
+    attributes:
+    - key: natural
+      constantValue: water
+      minZoom: 0
+    - key: intermittent
+      constantValue: 'true'
+      dataType: bool
+      includeWhen:
+        tag:
+          key: intermittent
+          value:
+          - true
+    - key: name
+      tagValue: name
+      dataType: string
+      includeWhen:
+        minTileCoverSize: 0.01
+      excludeWhen:
+        tag:
+          key: water
+          value:
+          - river
+          - canal
+          - stream
+  - sources:
+    - water_polygons
+    zoom:
+      minZoom: 0
+      minZoom: 14
+    includeWhen:
+      geometry: polygon
+    attributes:
+    - key: natural
+      constantValue: water
+      minZoom: 0
+  - sources:
+    - osm
+    zoom:
+      minZoom: 7
+      maxZoom: 14
+    includeWhen:
+      geometry: linestring
+      tag:
+        key: waterway
+        value:
+        - river
+        - stream
+        - canal
+    attributes:
+    - key: waterway
+      tagValue: waterway
+    - key: intermittent
+      constantValue: 'true'
+      dataType: bool
```

Comment:
The idea is to normalize different values e.g. "true" vs "yes", similar to what imposm3 does in OpenMapTiles.  I'm not sure how real of a problem that actually is in the database though...

Reply:
Thinking more about this, it would probably be easier to drop data typing and let the user specify yes/true as a value list in cases where this problem is real.

Reply:
Yeah, I was thinking that with yaml or json this is the difference between `true` and `"true"` or `1` and `"1"` - the main issue would be parsing values from osm tags as boolean vs. number

Reply:
Resolved in b0f3d60